RELEASES.md

```diff
@@ -22,172 +26,230 @@ Below are the versioning schemes used for publishing debs:
+        `1.1~18.04.1-rc1`.
+
+    f. For each release, dput to the staging PPA:
+      * `dput ppa:ua-client/staging ../out/<package_name>_source.changes`
+      * After each `dput` wait for the "Accepted" email from Launchpad before moving on.
+
+### II. Release to Ubuntu (devel and SRU)
+
+> Note: `impish` is used throughout as a reference to the current devel release. This will change.
+
+1. Prepare SRU Launchpad bugs.
+
+    a. We do this even before a succesful merge into ubuntu/devel because the context added to these bugs is useful for the Server Team reviewer.
+
+    b. Create a new bug on Launchpad for ubuntu-advantage-tools and use the format defined [here](https://wiki.ubuntu.com/UbuntuAdvantageToolsUpdates#SRU_Template) for the description.
+      * The title should be in the format `[SRU] ubuntu-advantage-tools (27.1 -> 27.2) Xenial, Bionic, Focal, Hirsute`, substituting version numbers and release names as necessary.
+    
+    c. For each Launchpad bug fixed by this release (which should all be referenced in our changelog), add the SRU template to the description and fill out each section.
+      * Leave the original description in the bug at the bottom under the header `[Original Description]`.
+      * For the testing steps, include steps to reproduce the bug. Then include instructions for adding `ppa:ua-client/staging`, and steps to verify the bug is no longer present.
+
+2. Set up the Merge Proposal (MP) for ubuntu/devel
+
+    a. `git-ubuntu clone ubuntu-advantage-tools; cd ubuntu-advantage-tools`
+
+    b. `git remote add upstream git@github.com:canonical/ubuntu-advantage-client.git`
+
+    c. `git fetch upstream`
+
+    d. `git rebase --onto pkg/ubuntu/devel <last-version-tag> <this-version-tag>`
+      * e.g. `git rebase --onto pkg/ubuntu/devel 27.0.2 27.1`
+      * You may need to resolve conflicts, but hopefully these will be minimal.
+      * You'll end up in a detached state
+
+    e. `git checkout -B upload-<this-version>-impish`
+      * This creates a new local branch name based on your detached branch
+
+    f. Make sure the changelog version contains the release version in the name (For example, `27.1~21.10.1`)
+
+    g. `git push <your_launchpad_user> upload-<this-version>-impish`
+
+    h. On Launchpad, create a merge proposal for this version which targets `ubuntu/devel`
+      * For an example, see the [27.0.2 merge proposal](https://code.launchpad.net/~chad.smith/ubuntu/+source/ubuntu-advantage-tools/+git/ubuntu-advantage-tools/+merge/402459)
+      * Add 2 review slots for `canonical-server` and `canonical-server-core-reviewers`
+3. Set up the MP for past Ubuntu releases based on the ubuntu/devel PR
+
+    a. Create a PR for each target series based off your local `release-${UA_VERSION}-impish` branch:
+    ```bash
+    UA_VERSION=<UA-VERSION>
+    SRU_BUG=<SRU-BUG>
+    LP_USER=<LAUNCHPAD-USERNAME>
+
+    for release in xenial bionic focal groovy hirsute
+    do
+      rm -rf ../out
```

Comment:
since we are not running `build-package` as part of this script, I think we can drop this line

Reply:
Good catch! Fixed now